release-calendar/src/client/components/EventCard.tsx

```diff
@@ -0,0 +1,160 @@
+      emojiName: 'experiment',
+    },
+    {
+      channel: Channel.STABLE,
+      text: 'Promoted to Stable on ',
+      title: 'Stable',
+      emoji: '🏠',
+      emojiName: 'house',
+    },
+    {
+      channel: Channel.LTS,
+      text: 'Promoted to LTS on ',
+      title: 'Long Term Stable',
+      emoji: '🏙️',
+      emojiName: 'city',
+    },
+  ];
+
+  render(): JSX.Element {
+    return (
+      <div className={'event-card'}>
+        <div className={this.props.eventApi.classNames[0]}>
+          <div className={'event-top'}></div>
+          <div className={'event-content'}>
+            <div className='content-row'>
+              <div className={'big-emoji'}>
+                <span role='img' aria-label={'train'}>
+                  {'🚄'}
+                </span>
+              </div>
+              <h2
+                className={
+                  'title-row'
+                }>{`Release ${this.props.eventApi.title}`}</h2>
+            </div>
+            <div className='content-content'>
+              {/* TODO: uncomment below when cherrypick is connected 
+              (will be adding cherrypick attribute to releaseDates) */}
+              {/* {this.state.releaseDates.cherrypick && (
+                <div className={'content-row'}>
+                  <div className={'emoji'}>
+                    <span role='img' aria-label={'flower'}>
+                      {'🌸'}
+                    </span>
+                  </div>
+                  <div className={'text'}>{'Cherrypick!'}</div>
+                </div>
+              )} */}
+              <div className={'content-row'}>
+                <div className={'emoji'}>
+                  <span role='img' aria-label={'laptop'}>
+                    {'🖥'}
+                  </span>
+                </div>
+                <div className={'text'}>
+                  <a
+                    href={`https://github.com/ampproject/amphtml/releases/tag/${this.props.eventApi.title}`}
+                    target='_blank'
+                    rel='noopener noreferrer'>
+                    {'Github Release'}
+                  </a>
+                </div>
+              </div>
+              <h3 className={'subtitle-row'}>{'Schedule'}</h3>
+              {this.state.releaseDates != null &&
+                this.state.releaseDates.dates.map((row) => {
+                  const match = this.schedule.find(
+                    (type) => type.channel == row.channel,
+                  );
+                  return (
+                    match && (
+                      <div className={'content-row'} key={match.emoji}>
+                        <div className={'emoji'}>
+                          <span role='img' aria-label={'row.emojiName'}>
+                            {match.emoji}
+                          </span>
+                        </div>
+                        <div className={'text'}>
+                          {match.text + moment(row.date).format('MMMM Do, hA')}
```

Comment:
Instead of having a `match.text` field, this can serve as a template to remove duplication.

Reply:
I'm not sure what you mean. Hm, if you're thinking that there could be something like
Promoted to Opt-in Experimental, Opt-In Beta on April 20th, 11 am
Promoted to Opt-in Experimental, Opt-In Beta on April 20th, 11 am
one line which represents `Channel.OPT_IN_EXPERIMENTAL` and the other `Channel.OPT_IN_BETA,` that wont happen because `Channel.OPT_IN_BETA` doesn't exist in the schedule template. There is the possibility of duplication if there had been a rollback and a release is getting reinstated to a channel it was already in. Is that what you're referring to? If so, I can brainstorm about adding a rollback line to divide the two promotions?

Reply:
By template I mean removing the `match.text` field and putting here:
`{match.title} was promoted ...`

You don't have to have a single line for experimental/beta, if it makes it easier. 1 channel per line is okay.